Classes/EelHelpers/AbstractImageSourceHelper.php

```diff
@@ -110,35 +149,35 @@ public function srcset($mediaDescriptors): string
 
             foreach ($descriptors as $descriptor) {
                 if (preg_match('/^(?<width>[0-9]+)w$/u', $descriptor, $matches)) {
-                    $width = (int) $matches['width'];
+                    $width = (int)$matches['width'];
                     $scaleFactor = $width / $this->getCurrentWidth();
                     $scaled = $this->scale($scaleFactor);
                     $srcsetArray[] = $scaled->src() . ' ' . $width . 'w';
-                } elseif (preg_match('/^(?<factor>[0-9\\.]+)x$/u', $descriptor, $matches)){
-                    $factor = (float) $matches['factor'];
+                } elseif (preg_match('/^(?<factor>[0-9\\.]+)x$/u', $descriptor, $matches)) {
+                    $factor = (float)$matches['factor'];
                     $scaled = $this->scale($factor);
                     $srcsetArray[] = $scaled->src() . ' ' . $factor . 'x';
                 }
             }
             return implode(', ', $srcsetArray);
-        } else {
-            return $this->src();
         }
+
+        return $this->src();
     }
 
     /**
-     * Define wich methods are available in the eel context
+     * Define which methods are available in the Eel context
      *
      * @param string $methodName
      * @return bool
      */
     public function allowsCallOfMethod($methodName)
     {
-        if (in_array($methodName, ['setWidth', 'setHeight', 'setDimensions', 'applyPreset', 'src', 'srcset'])) {
+        if (in_array($methodName, ['setWidth', 'setHeight', 'setDimensions', 'applyThumbnailPreset', 'useVariantPreset', 'src', 'srcset'])) {
```

Comment:
Since applyPreset is renamed both methods should use simular wording. I think 'useThumbnailPreset' and 'useVariantPreset' would be fine

Reply:
I uses `applyVariantPreset` first, but then realized that nothing is really "applied", since the variant is simply "used" as basis for further processing. The thumbnail preset OTOH is really "applied" in that it sets width and height. Thus the difference.

Then again, it "uses" the width and height from the thumbnail preset, as a base, so that's fine, too. Indeed. I'll adjust that to `use*Preset` for both.

Reply:
I agree to your decision, thanks for clarifying